extensions/resteasy-classic/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/QuarkusRestPathTemplateInterceptor.java

```diff
@@ -24,7 +26,13 @@
     @AroundInvoke
     Object restMethodInvoke(InvocationContext context) throws Exception {
         QuarkusRestPathTemplate annotation = getAnnotation(context);
-        if ((annotation != null) && (request.getCurrent() != null)) {
+        RoutingContext routingContext = null;
+        try {
+            routingContext = request.getCurrent();
+        } catch (ContextNotActiveException ex) {
```

Comment:
This is a nit you can ignore if you want, but some IDEs suggest you should name this variable "ignored" for clarity (yes, IDE linter, yes I am ignoring this on purpose, stop chiding me about it)...

Reply:
Ha, afaik my IDE only suggests that when the block is completely empty, adding a comment prevents that!